src/javascripts/auth.js

```diff
@@ -156,7 +156,7 @@ exports.loginRequired = function (callback, force) {
 				globalEvents.trigger('auth.loginRequired', {
 					callback: function (errExt) {
 						if (errExt) {
-							callback(errExt || new Error("Login failed."));
```

Comment:
Mind quickly explaining how this callback is called and why we always expect `errExt`.

Reply:
We always expect `errExt` because we are inside an `if` statement which has already proven `errExt` exists :-D

Reply:
Oh my God I'm so stupid.